src/Car.js

```diff
@@ -0,0 +1,31 @@
+import {
+  MOVE_CONDITION,
+  MOVE_COUNT,
+  START_POSITION,
+} from "./constants/setting.js";
+import { generateRandomNumber } from "./utils.js";
+
+export default class Car {
+  #name;
+  #position;
+
+  constructor(name) {
+    this.#name = name;
+    this.#position = START_POSITION;
+  }
```

Comment:
START_POSITION을 Car가 아닌 다른 곳에서 관리하는 이유가 있을까요?

```suggestion
  static START_POSITION = 0;
  constructor(name) {
    this.#name = name;
    this.#position = Car.START_POSITION;
  }
```

이렇게 Car 내부에 있어야 관리하기도 쉽고 맥락을 이해하기가 더 수월할 것 같아요!

Reply:
상수는 상수 파일에 저장해야 한다는 생각만 가지고 상수 파일에 작성했는데, JunilHwang께서 말씀하신 것처럼 Car 내부에서만 사용되니 Car 클래스 내부에 작성하는 것이 코드르 이해하기 쉽다고 생각이 드네요!!

감사합니다!!